app/views/api/v1/conversations/messages.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.messages @messages, partial: 'message', as: :message
```

Comment:
This file should be the index of messages, you can create a folder inside conversation or messages inside v1
Option 1: 
 `app/views/api/v1/conversations/messages/index.json.jbuilder `
Option 2:
`app/views/api/v1/messages/index.json.jbuilder`